models/User.js

```diff
@@ -19,4 +19,5 @@ const UserSchema = new Schema({
     default: Date.now,
   },
 });
-module.exports = User = mongoose.model("users", UserSchema);
+let User = mongoose.model("users", UserSchema);
```

Comment:
ESlint didn't like `User` being defined inline like that, so I made it stop yelling at me.